rskj-core/src/test/java/co/rsk/peg/BridgeSupportTestIntegration.java

```diff
@@ -3148,330 +3144,31 @@ void getRetiringFederationWallet_nonEmpty() throws IOException {
 
     @Test
     void getLockWhitelistMethods() {
```

Comment:
i think this test shouldnt be here either, but in `WhitelistSupportImplTest`. Is not testing anything related to `bridgeSupport`.

Reply:
This is not actually a Unit Test. It is an Integration Test. WDYT if we create a `WhitelistIT` class to put this test into it? @marcos-iov @julia-zack

Reply:
Done!